app/src/main/java/net/squanchy/schedule/ScheduleView.java

```diff
@@ -1,23 +1,36 @@
-package net.squanchy.schedule.view;
+package net.squanchy.schedule;
 
 import android.content.Context;
 import android.graphics.Typeface;
 import android.support.design.widget.CoordinatorLayout;
 import android.support.design.widget.TabLayout;
 import android.support.v4.view.ViewPager;
+import android.support.v7.widget.Toolbar;
 import android.util.AttributeSet;
+import android.view.LayoutInflater;
 import android.view.View;
+import android.view.ViewGroup;
 
 import net.squanchy.R;
 import net.squanchy.schedule.domain.view.Schedule;
+import net.squanchy.schedule.view.ScheduleViewPagerAdapter;
+
+import io.reactivex.android.schedulers.AndroidSchedulers;
+import io.reactivex.disposables.Disposable;
 
 import uk.co.chrisjenx.calligraphy.CalligraphyUtils;
 import uk.co.chrisjenx.calligraphy.TypefaceUtils;
 
 public class ScheduleView extends CoordinatorLayout {
 
+    public static ScheduleView inflate(Context context, ViewGroup parent) {
```

Comment:
I really really dislike this being here. Can you put it in a `BottomTabsPagesFactory` of some sort instead, that you can use in the adapter to inflate the views?